src/identifier.rs

```diff
@@ -114,7 +114,7 @@ mod interner {
     use typed_arena::Arena;
 
     /// A symbol is a correspondance between an [Ident](super::Ident) and its string representation stored in the [Interner].
-    #[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+    #[derive(Default, Debug, Clone, Copy, PartialEq, Eq, Hash)]
```

Comment:
Why do you need a default for `Symbol`? I fear it won't do what you expect. Symbols should be strictly generated by the string interner, they are indices inside an allocation table. Here deriving `Default` will give you index `0`, so this will probably just point to the first identifier ever allocated since Nickel started, which will probably not be empty. If you want `Ident` to implement `Default` (although it sounds a bit fishy, do you need that for tests?), I think you have to implement it by hand as `Ident::from("")` or something like that.

Reply:
Thanks, fixed!

Reply:
@gaga Did you push your changes? I still see a `Default` implementation for `Symbol` and `Ident`.